chart/k8gb/values.schema.json

```diff
@@ -600,6 +603,58 @@
             ],
             "title": "Route53"
         },
+        "AzureDNS": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "enabled": {
+                    "type": "boolean"
+                },
+                "createAuthSecret": {
+                    "type": "boolean",
+                    "default": true
+                },
+                "authSecretName": {
+                    "type": "string",
+                    "default": "external-dns-secret-azure"
+                },
+                "tenantId": {
+                    "type": "string",
+                    "minLength": 1
+                },
+                "subscriptionId": {
+                    "type": "string",
+                    "minLength": 1
+                },
+                "resourceGroup": {
+                    "type": "string",
+                    "minLength": 1
+                },
+                "aadClientId": {
+                    "type": "string"
+                },
+                "aadClientSecret": {
+                    "type": "string"
+                },
```

Comment:
As pointed out above, I have concerns about propagating `aadClientId` and `aadClientSecret` as plaintext values. Can we keep them in some input Secret? I appreciate that it might be tricky as we construct the external-dns Secret data in `external-dns.azure-credentials` template function.